sfcli/SmartFace.Cli/Commands/SubCamera/BaseCameraModifyingCmd.cs

```diff
@@ -26,6 +26,15 @@ public abstract class BaseCameraModifyingCmd
         [Option("-fd|--faceDetectorResourceId", "Face detector resource id for the camera", CommandOptionType.SingleValue)]
         public (bool HasValue, string Value) FaceDetectorResourceId { get; }
 
+        [Option("-pd|--pedestrianDetectorResourceId", "Pedestrian detector resource id for the camera", CommandOptionType.SingleValue)]
+        public (bool HasValue, string Value) PedestrianDetectorResourceId { get; }
+
+        [Option("-mp|--minPedestrianSize", "Minimum size of detected pedestrian in pixels (if >= 1) or relative (if > 0 && < 1)", CommandOptionType.SingleValue)]
```

Comment:
optional/suggestion - "or relative" - adding also what it is relative to?

Reply:
Updated the description